### PR TITLE
Add tutorial for Celery task setup.

### PR DESCRIPTION
**Pull request by n2ygk**
## Description of the Change
Document how to use Celery to automate clearing expired tokens.

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [x] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: docs/tutorial/tutorial_05.rst [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+.. code-block:: python
+
+    from celery import shared_task
+
+    @shared_task
+    def clear_tokens():
+	from oauth2_provider.models import clear_expired
+
+	clear_expired()
```
Comment (alexei) [inline docs/tutorial/tutorial_05.rst:88]:
```suggestion
    def clear_tokens():
		from oauth2_provider.models import clear_expired

		clear_expired()
```

Comment (alexei) [reply docs/tutorial/tutorial_05.rst:88]: This is how it shows for me:

Screenshot 2022-03-21 at 17 41 19

I thought the indentation was broken. If it's not, please ignore my comment. Maybe it's just a GitHub rendering issue.

Comment (alexei) [reply docs/tutorial/tutorial_05.rst:88]: I see the file [shows up just fine](https://github.com/jazzband/django-oauth-toolkit/blob/fde14c0566606e4cf4c20ad73981549f0594a39e/docs/tutorial/tutorial_05.rst), so please ignore my comment.

Comment (n2ygk) [reply docs/tutorial/tutorial_05.rst:88]: @alexei go figure, github doesn't render RST properly;-)

If you are able to approve the PR then I can merge it. I'm not sure whether only Jazzband members can be approvers though -- github wouldn't let me request you as a reviewer....

Comment (alexei) [reply docs/tutorial/tutorial_05.rst:88]: Yeah, that's alright, it serves mostly those who are not aware of it. We've already upgraded to 1.7.1 and addressed the issue internally 👍

File: docs/tutorial/tutorial_05.rst [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+Now your Celery Beat and Celery Workers should start running the task every 10 seconds.
+
+The Beat console will look like this:
+
+::
+
+   [2022-03-19 22:06:35,605: INFO/MainProcess] Scheduler: Sending due task clear stale tokens (tutorial.tasks.clear_tokens)
+
+And the Workser console like this:
```
Comment (Andrew-Chen-Wang) [inline docs/tutorial/tutorial_05.rst:149]:
```suggestion
And the Workers console like this:
```

File: docs/tutorial/tutorial_05.rst [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+.. image:: admin+celery.png
+  :width: 500
+  :alt: Django Admin interface screenshot
+
+Now let's define a fairly short (10 second) interval. Go to: http://127.0.0.1:8000/admin/django_celery_beat/intervalschedule/
+and select Add Interval, set number of intervals to 10 and interval period to seconds and Save.
+
+Then go to http://127.0.0.1:8000/admin/django_celery_beat/periodictask/ to add a new periodic task by
+selecting `Add Periodic Task <http://127.0.0.1:8000/admin/django_celery_beat/periodictask/add/>`_ and
+select ``tutorial.tasks.clear_tokens`` and chose the ``every 10 seconds`` interval schedule and Save.
```
Comment (Andrew-Chen-Wang) [inline docs/tutorial/tutorial_05.rst:134]:
```suggestion
select ``tutorial.tasks.clear_tokens``, choose the ``every 10 seconds`` interval schedule, and "Save."
```

File: docs/tutorial/tutorial_05.rst [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+Add Celery
+----------
+Make sure you virtualenv is active and install `celery` and
+`django-celery-beat <https://django-celery-beat.readthedocs.io/>`_.
+
+::
+
+    pip install celery django-celery-beat
+
+Update your list of installed apps to include both your AS server app -- we'll call it ``tutorial``,
```
Comment (Andrew-Chen-Wang) [inline docs/tutorial/tutorial_05.rst:48]: AS server app?

File: docs/tutorial/tutorial_05.rst [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+`Celery <https://docs.celeryq.dev/>`_?
+
+Set up RabbitMQ
+---------------
+Celery components communicate via a message queue. We'll use `RabbitMQ <https://www.rabbitmq.com/>`_.
+
+Install Rabbit MQ on MacOS
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you are using MacOS it's likely you are already using `Homebrew <https://brew.sh/>`_. If not, now's
+the time to start.
```
Comment (Andrew-Chen-Wang) [inline docs/tutorial/tutorial_05.rst:18]:
```suggestion
the time to install this fantastic package manager.
```

File: docs/tutorial/tutorial_05.rst [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+In :doc:`Part 1 <tutorial_01>` you created your own :term:`Authorization Server` and it's running along just fine.
+However, the database is getting cluttered with expired tokens. You can periodically run
+the :doc:`cleartokens management command <../management_commands>`, but why not automate this with
+`Celery <https://docs.celeryq.dev/>`_?
+
+Set up RabbitMQ
+---------------
+Celery components communicate via a message queue. We'll use `RabbitMQ <https://www.rabbitmq.com/>`_.
+
+Install Rabbit MQ on MacOS
```
Comment (Andrew-Chen-Wang) [inline docs/tutorial/tutorial_05.rst:15]:
```suggestion
Install RabbitMQ on MacOS
```

**Review by Andrew-Chen-Wang — CHANGES_REQUESTED** [review]

File: docs/tutorial/tutorial_05.rst (hunk 4 above)
Comment (n2ygk) [reply docs/tutorial/tutorial_05.rst:48]: Yeah, a server server :-)

**Comment by n2ygk** [comment]
@Andrew-Chen-Wang looks like my lack of github skills requires you to review/approve again.

**Review by Andrew-Chen-Wang — APPROVED** [review]
haha np

**Pull request merged**